submissions/sprint_b/test_helpers.py

```diff
@@ -0,0 +1,76 @@
+from sprint_b.helpers import FileManager as Fm
+from sprint_b.helpers import Vocabulary as Vocab
+from sprint_b.helpers import EpithetGenerator as Epgen
+import pytest
+from pathlib import PurePath as Path
+import os
+p = Path(os.getcwd())
+project_root = str(p.parent.parent)
```

Comment:
- project root should be a constant, constants are capitalized
- it's better to define project root in one location and import it